<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1013</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StdpC 2016 RC2a</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="HHBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>320</y>
      <width>731</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Ionic conductances</string>
    </property>
    <widget class="ComponentTable" name="currentTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>16</y>
       <width>631</width>
       <height>124</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>113</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>101</number>
     </attribute>
    </widget>
    <widget class="QPushButton" name="HHActivate">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>16</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Activate all</string>
     </property>
    </widget>
    <widget class="QPushButton" name="HHDeactivate">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>46</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Deactivate all</string>
     </property>
    </widget>
    <widget class="QPushButton" name="HHClear">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>76</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear inactive</string>
     </property>
    </widget>
    <widget class="QPushButton" name="HHReset">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>106</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="SynBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>170</y>
      <width>731</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Synapses</string>
    </property>
    <widget class="ComponentTable" name="synapseTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>16</y>
       <width>631</width>
       <height>124</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>113</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>101</number>
     </attribute>
    </widget>
    <widget class="QPushButton" name="SynReset">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>106</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SynClear">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>76</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear inactive</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SynDeactivate">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>46</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Deactivate all</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SynActivate">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>16</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Activate all</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="StartBut">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StopBut">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QListWidget" name="MessageWindow">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>251</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="DAQBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>731</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Data sources</string>
    </property>
    <widget class="DaqTable" name="DAQTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>16</y>
       <width>631</width>
       <height>124</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>143</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>101</number>
     </attribute>
    </widget>
    <widget class="QPushButton" name="DAQReset">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>46</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DAQClear">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>16</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear inactive</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1013</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Protocol"/>
    <addaction name="actionSave_Protocol"/>
    <addaction name="actionLoad_Script"/>
    <addaction name="actionUnload_Script"/>
    <addaction name="actionExport_Log"/>
    <addaction name="actionClear_Log"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuConfig">
    <property name="title">
     <string>Config</string>
    </property>
    <addaction name="actionData_saving"/>
    <addaction name="actionData_display"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConfig"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionConfig">
   <property name="text">
    <string>Config</string>
   </property>
  </action>
  <action name="actionLoad_Protocol">
   <property name="text">
    <string>Load Protocol</string>
   </property>
  </action>
  <action name="actionSave_Protocol">
   <property name="text">
    <string>Save Protocol</string>
   </property>
  </action>
  <action name="actionLoad_Script">
   <property name="text">
    <string>Load Script</string>
   </property>
  </action>
  <action name="actionUnload_Script">
   <property name="text">
    <string>Unload Script</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionDAQ">
   <property name="text">
    <string>DAQ</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About StdpC</string>
   </property>
  </action>
  <action name="actionInput_channels">
   <property name="text">
    <string>Input channels</string>
   </property>
  </action>
  <action name="actionOutput_channels">
   <property name="text">
    <string>Output channels</string>
   </property>
  </action>
  <action name="actionSave_config">
   <property name="text">
    <string>Save config</string>
   </property>
  </action>
  <action name="actionExport_Log">
   <property name="text">
    <string>Export Log</string>
   </property>
  </action>
  <action name="actionClear_Log">
   <property name="text">
    <string>Clear Log</string>
   </property>
  </action>
  <action name="actionElectrode_setup">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Electrode setup</string>
   </property>
  </action>
  <action name="actionData_saving">
   <property name="text">
    <string>Data saving</string>
   </property>
  </action>
  <action name="actionHH_models">
   <property name="text">
    <string>Hodgkin-Huxley models</string>
   </property>
  </action>
  <action name="actionData_display">
   <property name="text">
    <string>Data display</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComponentTable</class>
   <extends>QTableWidget</extends>
   <header>ComponentTable.h</header>
  </customwidget>
  <customwidget>
   <class>DaqTable</class>
   <extends>QTableWidget</extends>
   <header>DaqTable.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>StartBut</tabstop>
  <tabstop>StopBut</tabstop>
  <tabstop>MessageWindow</tabstop>
  <tabstop>DAQTable</tabstop>
  <tabstop>DAQClear</tabstop>
  <tabstop>DAQReset</tabstop>
  <tabstop>synapseTable</tabstop>
  <tabstop>SynActivate</tabstop>
  <tabstop>SynDeactivate</tabstop>
  <tabstop>SynClear</tabstop>
  <tabstop>SynReset</tabstop>
  <tabstop>currentTable</tabstop>
  <tabstop>HHActivate</tabstop>
  <tabstop>HHDeactivate</tabstop>
  <tabstop>HHClear</tabstop>
  <tabstop>HHReset</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
